tests/rptest/tests/wasm_topics_test.py

```diff
@@ -47,39 +47,3 @@ def rpk_partition_sort(p):
             if output != test_cfg:
                 raise Exception("Bad config, expected: %s observed: %s" %
                                 (test_cfg, output))
-
-
-class WasmDeleteTopicsTest(WasmIdentityTest):
-    def __init__(self, test_context, num_records=10, record_size=1024):
-        super(WasmDeleteTopicsTest, self).__init__(test_context,
-                                                   extra_rp_conf=None,
-                                                   num_records=num_records,
-                                                   record_size=record_size)
-
-    def wasm_test_plan(self):
-        return [
-            WasmScript(inputs=self.wasm_test_input(),
-                       outputs=["output_1", "output_2", "output_3"],
-                       script=WasmTemplateRepository.IDENTITY_TRANSFORM)
-        ]
-
-    @ok_to_fail  # https://github.com/redpanda-data/redpanda/issues/3745
-    @cluster(num_nodes=4)
-    def verify_materialized_topics_test(self):
```

Comment:
I'm concerned about the gap left in test coverage after this commit is merged. Should we document somewhere that we need a WasmDeleteTopics test in the future?

Reply:
I left the explanation [in the commit message](https://github.com/redpanda-data/redpanda/pull/4557/commits/964bb41bebc9aa60acb629ab8311de075bd87d12), is there somewhere else better we could put this? Maybe a link in the source?

Reply:
Fair point.

Reply:
I think when we re-pick up the WASM workstream, we will want to look at what all tests are required. That said, I wonder if we leave such tests...commented out rather than delete?

Reply:
I specifically deleted it as stated in the commit because it was working as expected. Deleting materialized topics while coprocessors are still processing inputs may re-create said topics